ipam/azure.go

```diff
@@ -172,7 +173,9 @@ func (s *azureSource) refresh() error {
 					log.Printf("[ipam] Failed to create address:%v err:%v.", address, err)
 					continue
 				}
+				addesscount++
 			}
+			log.Printf("[ipam] got %d addresses from interface %s, subnet %s", ifName, subnet)
```

Comment:
Could also reject pools with empty address lists in setaddressspace below.

Reply:
maybe log error

Reply:
Punting on this since nobody asked for it.